src/renderer/component/uriIndicator/view.jsx

```diff
@@ -1,7 +1,7 @@
 // @flow
 import React from 'react';
 import Button from 'component/button';
-import { buildURI } from 'lbryURI';
+import { buildURI } from 'lbry-redux';
 import classnames from 'classnames';
 // import Icon from 'component/common/icon';
```

Comment:
I know this was leftover from me, but this can just be deleted

Reply:
This is actually used on line 62 so we can leave it in.
`channelLink = link ? buildURI({ channelName, claimId: channelClaimId }, false) : false;`